packages/messaging/e2e/messaging.e2e.js

```diff
@@ -142,7 +142,7 @@ describe('messaging()', function () {
 
   describe('getInitialNotification', function () {
     it('returns null when no initial notification', async function () {
-      should.equal(await firebase.messaging().getInitialNotification(), null);
+      should.strictEqual(await firebase.messaging().getInitialNotification(), null);
```

Comment:
Nice! So this was a false-positive test then, where the value undefined was being coerced by javascript to equal null, but was *actually* `undefined` and thus not meeting the typescript type expectation correctly?

Also, you may not have it handy but I tried to find documentation on what objective-c types map specifically to what javascript types for return values and I could not find one! I only found javascript to native, not the other way around. If you know of any documentation I would love to see it

But this looks good, thank you very much for posting it and choosing a non-breaking style of repair

Reply:
No, I could not find any doc regarding this and I found this when I was running it on iOS, however if I remember correctly `nil` maps to `undefined` and `NSNull` maps to `null`.